magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -73,32 +72,32 @@ class NavigationDelegate(
     activity = null
   }
 
-  fun goTo(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, GO)
+  fun goTo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: DefaultTransition())
   }
 
-  fun show(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, SHOW)
+  fun show(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: ShowTransition())
   }
 
-  fun replaceAndGo(nextNavigableCompat: NavigableCompat) {
-    replace(nextNavigableCompat, GO)
+  fun replaceAndGo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    replace(nextNavigableCompat, overrideTransition ?: DefaultTransition())
   }
 
-  fun replaceAndShow(nextNavigableCompat: NavigableCompat) {
-    replace(nextNavigableCompat, SHOW)
+  fun replaceAndShow(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    replace(nextNavigableCompat, overrideTransition ?: ShowTransition())
   }
 
-  private fun replace(nextNavigableCompat: NavigableCompat, navType: NavigationType) {
+  private fun replace(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
```

Comment:
This is the same as `replaceAndGo` now, right? Could we remove the redundant one? (or better yet, remove `replaceAndGo` and `replaceAndShow` altogether?)